code/modules/research/stock_parts.dm

```diff
@@ -145,78 +147,78 @@ If you create T5+ please take a pass at gene_modder.dm [L40]. Max_values MUST fi
 	rating = 2
 	materials = list(MAT_METAL=80)
 
-//Rating 3
+//Tier 3
 
 /obj/item/stock_parts/capacitor/super
 	name = "super capacitor"
 	desc = "A super-high capacity capacitor used in the construction of a variety of devices."
 	icon_state = "super_capacitor"
-	rating = 3
+	rating = 4
```

Comment:
why is this all changed to 1,2,4,8 again?

Reply:
Somehow works in the bad deathclaw so probably should work somehow :ok_hand: here